docs/nlu/choosing-a-pipeline.rst

```diff
@@ -273,195 +301,7 @@ one component can calculate feature vectors for the training data, store
 that within the context and another component can retrieve these feature
 vectors from the context and do intent classification.
 
-Initially the context is filled with all configuration values, the arrows
+Initially the context is filled with all configuration values. The arrows
 in the image show the call order and visualize the path of the passed
 context. After all components are trained and persisted, the
 final context dictionary is used to persist the model's metadata.
-
-
-The "entity" object explained
------------------------------
-After parsing, the entity is returned as a dictionary.  There are two fields that show information
-about how the pipeline impacted the entities returned: the ``extractor`` field
-of an entity tells you which entity extractor found this particular entity, and
-the ``processors`` field contains the name of components that altered this
-specific entity.
-
-The use of synonyms can also cause the ``value`` field not match the ``text``
-exactly. Instead it will return the trained synonym.
-
-.. code-block:: json
-
-    {
-      "text": "show me chinese restaurants",
-      "intent": "restaurant_search",
-      "entities": [
-        {
-          "start": 8,
-          "end": 15,
-          "value": "chinese",
-          "entity": "cuisine",
-          "extractor": "CRFEntityExtractor",
-          "confidence": 0.854,
-          "processors": []
-        }
-      ]
-    }
-
-.. note::
-
-    The ``confidence`` will be set by the ``CRFEntityExtractor`` and ``DIETClassifier`` component. The
-    ``DucklingHTTPExtractor`` will always return ``1``. The ``SpacyEntityExtractor`` extractor does not provide this
-    information and returns ``null``.
-
-
-
-Pipeline Templates (deprecated)
```

Comment:
Shouldn't we keep this section until those pipeline templates are removed in Rasa 2.0?